pysd/py_backend/utils.py

```diff
@@ -29,8 +35,8 @@ def dict_find(in_dict, value):
     'Key2'
 
     """
-    # Todo: make this robust to repeated values
-    # Todo: make this robust to missing values
+    # TODO make this robust to repeated values
```

Comment:
Do we have an open issue for this todo?

Reply:
I think we haven't, I will open one, we need to explore how this may fail.

Reply:
I have removed this functions, as it is not used by any other function and not added to the models file. We are using  find_subscript_name instead

Reply:
I thought this was used for looking things up in the namespace dictionary, but we may have moved away from that at some point. Good to identify unused code.